docs/ref/class-based-views/generic-date-based.txt

```diff
@@ -254,19 +254,19 @@ views for displaying drilldown pages for date-based data.
 
     **Example myapp/urls.py**::
 
-        from django.conf.urls import url
+        from django.urls import path
 
         from myapp.views import ArticleMonthArchiveView
 
         urlpatterns = [
-            # Example: /2012/aug/
-            url(r'^(?P<year>[0-9]{4})/(?P<month>[-\w]+)/$',
-                ArticleMonthArchiveView.as_view(),
-                name="archive_month"),
             # Example: /2012/08/
-            url(r'^(?P<year>[0-9]{4})/(?P<month>[0-9]+)/$',
+            path('<int:year>/<int:month>/',
```

Comment:
This crashes because the type of kwargs['year']/[''month'] that's passed to the view is changed from string to int:
```
  File "/home/tim/code/django/django/views/generic/dates.py", line 617, in _date_from_string
    datestr = year + delim + month + delim + day
TypeError: unsupported operand type(s) for +: 'int' and 'str'
```
This particular case can be fixed by adding `str()` casting in the generic views but generally this could be a tricky point in adapting to the new system as views can no longer assume the view args/kwargs are `str`. At least worth mentioning in the release notes, I think.